bot/exts/moderation/stream.py

```diff
@@ -173,6 +175,46 @@ async def revokestream(self, ctx: commands.Context, member: discord.Member) -> N
         await ctx.send(f"{Emojis.cross_mark} This member doesn't have video permissions to remove!")
         log.debug(f"{member} ({member.id}) didn't have the streaming permission to remove!")
 
+    @commands.command(aliases=('lstream',))
+    @commands.has_any_role(*MODERATION_ROLES)
+    async def liststream(self, ctx: commands.Context) -> None:
+        """Lists all non-staff users who have permission to stream."""
+        non_staff_members_with_stream = [
+            member
+            for member in ctx.guild.get_role(Roles.video).members
+            if not any(role.id in STAFF_ROLES for role in member.roles)
+        ]
+
+        # List of tuples (UtcPosixTimestamp, str)
+        # This is so that output can be sorted on [0] before passed it's to the paginator
+        streamer_info = []
+        for member in non_staff_members_with_stream:
+            if revoke_time := await self.task_cache.get(member.id):
+                # Member only has temporary streaming perms
+                revoke_delta = Arrow.utcfromtimestamp(revoke_time).humanize()
+                message = f"{member.mention} will have stream permissions revoked {revoke_delta}."
+            else:
+                message = f"{member.mention} has permanent streaming permissions."
+
+            # If revoke_time is None use max timestamp to force sort to put them at the end
+            streamer_info.append(
+                (revoke_time or Arrow.max.timestamp(), message)
+            )
+
+        if streamer_info:
+            # Sort based on duration left of streaming perms
+            streamer_info.sort(key=itemgetter(0))
+
+            # Only output the message in the pagination
+            lines = [line[1] for line in streamer_info]
+            embed = discord.Embed(
+                title=f"Members who can stream (`{len(lines)}` total)",
```

Comment:
I find the wording a bit weird but I really don't mind either way haha
```suggestion
                title=f"Members with streaming permission (`{len(lines)}` total)",
```

Reply:
I think something like 
```suggestion
                title=f"Found `{len(lines)}` members with streaming permission",
```